support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -38,7 +41,13 @@ class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
     services.acquisitionService.submit(
       SendAcquisitionEventStateAndRequestInfo(state, requestInfo)
     ).fold(
-      errors => throw AnalyticsServiceErrorList(errors), _ => HandlerResult(Unit, requestInfo)
+      errors => throw AnalyticsServiceErrorList(errors),
+      _ => {
+        val cloudwatchEvent = paymentSuccessRequest(Configuration.stage, paymentProviderFromPaymentMethod(state.paymentMethod), state.product)
+        AwsCloudWatchMetricPut(client)(cloudwatchEvent)
```

Comment:
no contributions is a P1 issue. however no analytics is not a P1 issue.  So if we want to use this for detecting p1 issues we should probably have a separate alarm for it.  If not, we should make it more clear in the alarm description that it could just be analytics that's failing.

Reply:
This was a good point - I have moved the cloudwatch request outside of the `future.fold` because we want it to happen regardless of whether the analytics request succeeds